ballerina-shell/modules/shell-cli/src/main/java/io/ballerina/shell/cli/jline/JlineSimpleCompleter.java

```diff
@@ -68,7 +71,7 @@ public JlineSimpleCompleter() {
     @Override
     public void complete(LineReader reader, ParsedLine line, List<Candidate> candidates) {
         if (hasPropertyPrefix(line, COMMAND_HELP)) {
-            if (line.wordIndex() == 1) {
+            if (line.wordIndex() == 1 && this.topicsCompleter != null) {
```

Comment:
This null check is not required right?

Reply:
Removed in 0fab9b14280a9b53f5846882922678a3f6d7ccc7